tezos/context/src/kv_store/in_memory.rs

```diff
@@ -414,6 +409,130 @@ impl InMemory {
         })
     }
 
+    /// Reload context from disk
+    fn reload_database(&mut self) -> Result<(), ReloadError> {
+        let (tree, parent_hash, commit) = {
+            let mut ondisk = Persistent::try_new(PersistentConfiguration {
+                db_path: Some("/tmp/tezedge/context".to_string()),
```

Comment:
@sebastiencs this could come from somewhere else right? it is hardcoded right now